app/styles/components/_session-offering.scss

```diff
@@ -0,0 +1,7 @@
+.plus-button, .minus-button {
+  outline: none;
+}
+
+.offering-section-top {
+  overflow: hidden;
```

Comment:
Its children are floats so I want that container to cover all of its children. The default behavior for parent containing floating children is that it will be not visible when you look for it.